.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: python
+python:
+  - "2.7"
+
+dist: trusty
+
+services:
+  - postgresql
```

Comment:
You're right, this is not really useful. With service postgresql, I get errors : 
```
ERROR:  could not access file "$libdir/postgis-2.3": No such file or directory
```
So I remove it, reinstall, and all works fine.